lib/flipper/configuration.rb

```diff
@@ -1,23 +1,23 @@
 module Flipper
   class Configuration
     def initialize
-      @default = -> { raise DefaultNotSet }
+      @default = -> { Flipper.new(Flipper::Adapters::Memory.new) }
     end
 
     # Controls the default instance for flipper. When used with a block it
     # assigns a new default block to use to generate an instance. When used
     # without a block, it performs a block invocation and returns the result.
     #
     #   configuration = Flipper::Configuration.new
-    #   configuration.default # => raises DefaultNotSet error.
+    #   configuration.default # => Flipper::DSL instance using Memory adapter
     #
-    #   # sets the default block to generate a new instance using Memory adapter
+    #   # sets the default block to generate a new instance using ActiveRecord adapter
     #   configuration.default do
-    #     require "flipper/adapters/memory"
-    #     Flipper.new(Flipper::Adapters::Memory.new)
+    #     require "flipper-active-record"
```

Comment:
```suggestion
    #     require "flipper-active_record"
```

Ugh dashes and underscores, amirite?